### ISSUE TITLE
A problem regarding some middle experiements in your paper

### ISSUE THREAD

**Issue by rxqy** [NieXC/pytorch-mula#2]
Hi, I really enjoy your job and your writing. However, I have some problems regarding the Table 1  in your paper, with the title 

> VGG16-FCN  based ablation studies on LIP validation set.

I understand that VGG16-FCN-MuLA is your proposed method here.

For VGG16-FCN, is it training two VGG-FCN models separately for the two tasks?

For VGG16-FCN-MTL, is it like
`input -> VGG16 -> pose and parsing loss`?

For VGG16-FCN-Add/Multi/Concat, is it add/multi/concate features from two parallel models? 
like
`input -> VGG16 -> Feature_pose  -- concate with parsing feature ->  pose loss `
`input -> VGG16 -> Feature_parse --  concate with pose feature -> parsing loss ` ?
Did you use any special tricks to train the network? Or just train with RMSprop, like you did it here?
Cause I followed my understandings here, and found a performance drop when compared to the VGG16-FCN model.

Sorry for the inconvenience,  and many thanks

**Comment by 498936103** [comment]
> Hi, I really enjoy your job and your writing. However, I have some problems regarding the Table 1 in your paper, with the title
> 
> > VGG16-FCN  based ablation studies on LIP validation set.
> 
> I understand that VGG16-FCN-MuLA is your proposed method here.
> 
> For VGG16-FCN, is it training two VGG-FCN models separately for the two tasks?
> 
> For VGG16-FCN-MTL, is it like
> `input -> VGG16 -> pose and parsing loss`?
> 
> For VGG16-FCN-Add/Multi/Concat, is it add/multi/concate features from two parallel models?
> like
> `input -> VGG16 -> Feature_pose -- concate with parsing feature -> pose loss `
> `input -> VGG16 -> Feature_parse -- concate with pose feature -> parsing loss ` ?
> Did you use any special tricks to train the network? Or just train with RMSprop, like you did it here?
> Cause I followed my understandings here, and found a performance drop when compared to the VGG16-FCN model.
> 
> Sorry for the inconvenience, and many thanks

hello, I am also readding the paper. Can I contact with you by QQ or WeChat or mail?

**Comment by NieXC** [comment]
@rxqy 

Many thanks for your interesting and sorry for the late reply due to busy on CVPR ddl.

Your understanding about ablation cases based on VGG16 is correct.

There is no any trick for training the models. 

About the perform drop, which model do you mean drops the performance comparing with VGG16-FCN? MuLA or others ?

**Comment by rxqy** [comment]
Many thanks! I made a mistake during training.
@498936103 I think if you have any problems, you can post it here.
I'm very happy to share my experience with you.

**Comment by zzw1123** [comment]
@rxqy Hi, I am also confused about the ablation study implementation.
As you mentioned above, 
```
For VGG16-FCN-MTL, is it like
input -> VGG16 -> pose and parsing loss
```
I am wondering what the output is when  supervised by pose and parsing loss together? Is it pose or part result?

**Comment by rxqy** [comment]
@zzw1123 
Both of them.
The input is the shared feature ( 512 layers taking vgg as an example ).
Then we add two non-shared classifier head to the shared feature.
i.e, one with input channel 512, output channel 20 for parsing output,
the other one with input channel 512, output channel 16 for joint output.

**Comment by zzw1123** [comment]
@rxqy Got it...so the two classifier is supervised by its loss respectively.
Thank you so much!
